cmd/kubeadm/app/cmd/phases/upgrade/apply/preflight.go

```diff
@@ -68,8 +68,8 @@ func runPreflight(c workflow.RunData) error {
 
 	initCfg, client, ignorePreflightErrors := data.InitCfg(), data.Client(), data.IgnorePreflightErrors()
 
-	// First, check if we're root separately from the other preflight checks and fail fast.
-	if err := preflight.RunRootCheckOnly(ignorePreflightErrors); err != nil {
+	// First, run upgrade preflight checks and fail fast.
+	if err := preflight.RunUpgradeChecks(ignorePreflightErrors); err != nil {
```

Comment:
we want to run the root check only first and then run any upgrade checks
this is how it's done for init and join. we changed it recently
https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/cmd/phases/init/preflight.go#L64-L71